src/components/TransactionHistoryRows/components/AssetsTransfers.tsx

```diff
@@ -0,0 +1,62 @@
+import { Box, Circle, Stack, Text as CText } from '@chakra-ui/react'
+import { useMemo } from 'react'
+import { Amount } from 'components/Amount/Amount'
+import { useButtonStyles } from 'hooks/useButtonStyles/useButtonStyles'
+import type { Transfer } from 'hooks/useTxDetails/useTxDetails'
+import { bn, bnOrZero } from 'lib/bignumber/bignumber'
+import { fromBaseUnit } from 'lib/math'
+
+import { FALLBACK_PRECISION } from '../constants'
+
+type AssetsTransfersProps = {
+  compactMode?: boolean
+  transfers: Transfer[]
+  index: number
+}
+
+export const AssetsTransfers: React.FC<AssetsTransfersProps> = ({
+  index,
+  compactMode,
+  transfers,
+}) => {
+  const { bg: circleBgColor, color: circleColor } = useButtonStyles({
+    colorScheme: 'blue',
+    variant: 'ghost-filled',
+  })
+
+  const aggregatedFiatValue = useMemo(
+    () =>
+      transfers
+        .reduce((acc, transfer) => {
+          const precision = transfer.asset?.precision ?? FALLBACK_PRECISION
+          const cryptoValue = fromBaseUnit(transfer.value, precision)
+          return acc.plus(bnOrZero(cryptoValue).times(transfer.marketData.price))
+        }, bn(0))
+        .toString(),
+    [transfers],
+  )
+
+  return (
+    <Stack
+      alignItems='center'
+      key={index}
```

Comment:
Given we have multiple transfers in this component, kept the index as a key - while inconsistent, the boilerplaty logic to make a sane key is probably not worth is just for the sake of being consistent with not using a an index as a key